flake-modules/dev/launch-test.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+: "${NIXVIM_NIX_COMMAND:=nix}"
+if [[ -z $NIXVIM_SYSTEM ]]; then
+	NIXVIM_SYSTEM=$(nix eval --raw --impure --expr "builtins.currentSystem")
+fi
+
+help() {
+	cat <<EOF
+Usage: tests [-i/--interactive] [-h/--help] [-t/test <test name>]
+
+Options:
+	-i/--interactive: Pick interactively the tests
+	-h/--help: Display this help message and exit
+	-t/--test <test name>: Only launch the specified test (conflicts with -i)
+EOF
+}
+
+if ! OPTS=$(getopt -o "hit:" -l "help,interactive,test:" -- "$@"); then
+	echo "Invalid options" >&2
+	help
+	exit 1
+fi
+
+eval set -- "$OPTS"
+
+test_name=
+interactive=false
+
+while true; do
+	case "$1" in
+	-h | --help)
+		help
+		exit 0
+		;;
+	-i | --interactive)
+		interactive=true
+		shift
+		;;
+	-t | --test)
+		test_name=$2
+		shift 2
+		;;
+	*) break ;;
+	esac
+done
+
+if [[ $interactive = true && -n $test_name ]]; then
+	echo "Can't specify both --interactive and --test" >&2
+	exit 1
+fi
+
+test_list() {
+	nix flake show --json 2>/dev/null |
```

Comment:
I didn't try anything of this, but I'm sure that `nix flake show` will be very slow. Try this instead: `nix eval .#checks.x86_64-linux.tests.entries --apply builtins.attrNames --json`

Reply:
Yup indeed nix flake show takes multiple seconds, thanks I'll try that

Reply:
@traxys is this resolved?

Reply:
Yes